lib/dal/src/context.rs

```diff
@@ -1162,7 +1177,17 @@ async fn rebase(
         ),
         si_events::Actor::System,
     );
-
+    span.record(
+        "si.change_set.id",
+        rebase_request.to_rebase_change_set_id.to_string(),
+    );
+    span.record(
+        "si.workspace.pk",
+        tenancy
+            .workspace_pk()
+            .unwrap_or(WorkspacePk::NONE)
+            .to_string(),
+    );
```

Comment:
Oh this will be helpful!

Also... optional tenant id probably needs to get removed. Unrelated to this PR, just noticed it on line 1188.